### PR TITLE
feat(isBemEntityName): add `isBemEntityName` method

### LINKED ISSUE #64: isBemEntityName method

### PR DESCRIPTION
**Pull request by blond**
Resolved #64

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,6 +233,23 @@ name.valueOf();
...
+Parameter    | Type            | Description
+-------------|-----------------|-----------------------
+`entityName` | `BemEntityName` | The entity to check.
+
+```js
+const BemEntityName = require('@bem/entity-name');
+
+const entityName = new BemEntityName({ block: 'input' });
+
+BemEntityName.isBemEntityName(entityName); // true
```
Comment (tadatuta) [inline README.md]: And if it's not for some reason let's rename the method to `isInstanceOf`?

Comment (blond) [inline README.md]: see https://github.com/bem-sdk/bem-entity-name/pull/2

`instanceof` will not work with different versions of `BemEntityName`.

File: index.js [hunk 2]
```diff
@@ -256,4 +258,22 @@ module.exports = class BemEntityName {
...
+     * @example
+     * const BemEntityName = require('@bem/entity-name');
+     *
+     * const entityName = new BemEntityName({ block: 'input' });
+     *
+     * BemEntityName.isBemEntityName(entityName); // true
+     * BemEntityName.isBemEntityName({}); // false
+     */
+    static isBemEntityName(entityName) {
+        return entityName && entityName.__isBemEntityName__;
```
Comment (tadatuta) [inline index.js]:
```js
return this.name === entityName.constructor.name;
```

File: index.js [hunk 3]
```diff
@@ -256,4 +258,22 @@ module.exports = class BemEntityName {
...
+     * @returns {boolean} A Boolean indicating whether or not specified entity is instance of BemEntityName.
+     * @example
+     * const BemEntityName = require('@bem/entity-name');
+     *
+     * const entityName = new BemEntityName({ block: 'input' });
+     *
+     * BemEntityName.isBemEntityName(entityName); // true
+     * BemEntityName.isBemEntityName({}); // false
+     */
+    static isBemEntityName(entityName) {
```
Comment (tadatuta) [inline index.js]: and I still didn't get why not to call it `isInstanceOf`

**Pull request merged**